src/script/components/MessagesList/Message/ContentMessage/asset/VideoAsset.tsx

```diff
@@ -116,102 +128,102 @@ const VideoAsset: React.FC<VideoAssetProps> = ({
-              {videoPlaybackError ? (
-                <div className="video-asset__playback-error label-xs">{t('conversationPlaybackError')}</div>
-              ) : (
-                <>
-                  {transferState === AssetTransferState.UPLOAD_PENDING ? (
-                    <div className="asset-placeholder loading-dots" />
-                  ) : (
-                    <div
-                      className={cx('video-asset__controls', {
-                        'video-asset__controls--hidden': isVideoLoaded && hideControls,
-                      })}
-                    >
-                      <div className="video-asset__controls-center">
-                        {displaySmall ? (
-                          <MediaButton
-                            mediaElement={videoElement}
-                            asset={asset}
-                            play={onPlayButtonClicked}
-                            transferState={transferState}
-                            uploadProgress={uploadProgress}
-                          />
-                        ) : (
-                          <MediaButton
-                            mediaElement={videoElement}
-                            large
-                            asset={asset}
-                            play={onPlayButtonClicked}
-                            pause={onPauseButtonClicked}
-                            cancel={() =>
-                              transferState === AssetTransferState.UPLOADING ? cancelUpload() : asset.cancelDownload()
-                            }
-                            transferState={transferState}
-                            uploadProgress={uploadProgress}
-                          />
-                        )}
-                      </div>
-
-                      {isVideoLoaded && (
-                        <div className="video-asset__controls__bottom">
-                          <SeekBar
-                            className="video-asset__controls__bottom__seekbar"
-                            data-uie-name="status-video-seekbar"
-                            mediaElement={videoElement}
-                          />
-                          <span
-                            className="video-asset__controls__bottom__time label-xs"
-                            data-uie-name="status-video-time"
-                          >
-                            {formatSeconds(videoTimeRest)}
-                          </span>
-                        </div>
+              onPlaying={onVideoPlaying}
+              onTimeUpdate={() => videoElement && setVideoTimeRest(videoElement.duration - videoElement.currentTime)}
+              onLoadedMetadata={() =>
+                videoElement && setVideoTimeRest(videoElement.duration - videoElement.currentTime)
+              }
+              style={{
+                backgroundColor: videoPreview ? '#000' : '',
+                visibility: transferState === AssetTransferState.UPLOADING ? 'hidden' : undefined,
```

Comment:
Do smth like that :)
```suggestion
                {...(transferState === AssetTransferState.UPLOADING && { visibility: 'hidden'}),
```

Reply:
Also maybe move this transferState === AssetTransferState.UPLOADING to variable, you have used it multiple times :)

Reply:
What is the benefit of this method, I don't see any benefit, I personally find it harder to read 🤔

Reply:
If you use it you don't need to pass undefined, and for me it's better than passing undefined :)

Reply:
Fixed with `className` :)